File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    do
    {
        printf("Height: ");
        h = get_int();
    }
    while(h < 0 || h > 23);
    
    for (int i = 0; i < h; i++)
    {
        int spaces = 0;
        int hash = 0;
        int hash_two = 0;
        
        while (spaces < h - (1+i))
        {
            printf(" ");
            spaces++;
        }
        
        while(hash < 1 + i)
        {
            printf("#");
            hash++;
        }
        
        printf("  ");
        
        while(hash_two < 1 + i)
        {
            printf("#");
            hash_two++;
        }
        
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

while True:
    print("Height: ", end="")
    h = cs50.get_int();
    if h >= 0 or h < 24:
        break
    
for i in range(h):
    
    for spaces in range(h - (1 + i)):
        print(" ", end="")
        
    for hash in range(1 + i):
        print("#", end="")
        
    print("  ", end="")
    
    for hash_two in range(1 + i):
        print("#", end="")
    
    print("")
    